microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/SubsystemRemove.java

```diff
@@ -0,0 +1,19 @@
+package org.wildfly.extension.microprofile.opentracing.smallrye;
+
+import org.jboss.as.controller.AbstractRemoveStepHandler;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.dmr.ModelNode;
+import org.jboss.logging.Logger;
+
+class SubsystemRemove extends AbstractRemoveStepHandler {
+    private static final Logger log = Logger.getLogger(SubsystemAdd.class);
+    static final SubsystemRemove INSTANCE = new SubsystemRemove();
+
+    private SubsystemRemove() {
+    }
+
+    @Override
+    protected void performRuntime(OperationContext context, ModelNode operation, ModelNode model) {
+        log.info("Removing tracing subsystem");
```

Comment:
This should be a debug or trace message and should use the `TracingLogger.ROOT_LOGGER.debug("Removing tracing subsystem")`. It also seems a little odd to me that a remove handler doesn't really do anything. This likely should just use the `ReloadRequiredRemoveStepHandler.INSTANCE` for the remove handler making this obsolete.